person.ru

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- Please, remember that the requirements ask for:
Getters for @id, @name, and @age.
Setters for @name and @age.
In this case, the @id is getter and setter. Please fix this. 🐶